crates/bevy_a11y/src/lib.rs

```diff
@@ -0,0 +1,55 @@
+use std::num::NonZeroU128;
+
+pub use accesskit;
+use accesskit::{Node, NodeId};
+use bevy_app::Plugin;
+use bevy_derive::{Deref, DerefMut};
+use bevy_ecs::{
+    prelude::{Component, Entity},
+    system::Resource,
+};
+
+#[derive(Component, Clone, Default, Deref, DerefMut)]
+pub struct AccessibilityNode(pub Node);
+
+impl From<Node> for AccessibilityNode {
+    fn from(node: Node) -> Self {
+        Self(node)
+    }
+}
+
+pub trait AccessKitEntityExt {
```

Comment:
Is this trait 100% needed and also required to be in the public interface? Seems like it could be an internal utility function.

Reply:
Actually, with windows as entities now, it may be possible to make this an implementation detail. Hadn't looked at it in a while but initially it had to add 2 because 1 was reserved for the primary window ID. But with everything sharing the same ID space, I may be able to make it `pub(crate)`. If I can, I will.

Reply:
Unfortunately, this needs to be exposed for UI and other accessibility implementors to agree on an entity-to-AccessKit ID mapping. AccessKit IDs need to be non-zero so it just adds 1.

Reply:
Agreed: I think this is mandatory for bevy to define.